pipelines/concourse.yml

```diff
@@ -305,6 +306,24 @@ jobs:
   on_success: *fixed-concourse
   on_failure: *broke-concourse
 
+- name: containerd-integration
+  public: true
+  max_in_flight: 2
```

Comment:
Is this a magic number?

Reply:
oh good catch! that was straight from `testflight` - while I can see the desire to limit the possible concurrent runs to not have N of them, I can't come up with a reason why 2 rather than, say, 3.

```
ag -Q "max_in_flight"
pipelines/concourse.yml
311:  max_in_flight: 2
329:  max_in_flight: 2
353:  max_in_flight: 2
564:  max_in_flight: 1
802:  max_in_flight: 1
```

I'll go with 1 per the desire to limit, but not have an opinion on "how much" 👍